### DISCUSSION TITLE
Alternative to backend.mergedPeaks in v7 beta

### DISCUSSION THREAD

**Discussion by pranav-asthana** [katspaugh/wavesurfer.js#2842] [Q&A]
### Question

I have been using wavesurfer.backend.mergedPeaks to preprocess and store peaks data so that when I load my application there is minimum latency. With the new version I see that the backend has been deprecated. Can you please advice on how I can continue to preprocess and store peaks data.

Thanks!

**Comment by katspaugh** [comment]
You can get the decoded audio buffer via `getDecodedData()`
